checker/bin-devel/build.sh

```
@@ -8,13 +8,9 @@ set -e
 
 SCRIPT_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)"
 source "$SCRIPT_DIR"/clone-related.sh
-# Download Gradle and dependencies, retrying in case of network problems.
-# echo "NO_WRITE_VERIFICATION_METADATA=$NO_WRITE_VERIFICATION_METADATA"
-if [ -z "${NO_WRITE_VERIFICATION_METADATA+x}" ]; then
-  (date && TERM=dumb timeout 300 ./gradlew --write-verification-metadata sha256 help --dry-run < /dev/null > /dev/null 2>&1) \
-    || (sleep 1m && date && TERM=dumb timeout 300 ./gradlew --write-verification-metadata sha256 help --dry-run)
-fi
+IS_CI="$("$SCRIPT_DIR"/is-ci.sh)"
```

Shellcheck (0.10.0)

[warning] IS_CI appears unused. Verify use (or export if used externally).

(SC2034)